.github/workflows/tests.sh

```diff
@@ -8,6 +8,14 @@ verdi daemon start 4
 verdi -p test_${AIIDA_TEST_BACKEND} run .ci/test_daemon.py
 verdi daemon stop
 
+# Make sure
+# - omissions are taken from the top-level .coveragerc
+# - coverage is reported as XML and in terminal including
+#   the numbers/ranges of lines which are not covered,
+# - coverage results of multiple tests are collected,
```

Comment:
Are you sure this is the case?

Reply:
this is a description of the specified flags and they seem to be all respected based on the test output